api/cpp/include/slint.h

```diff
@@ -85,6 +85,14 @@ inline ItemRef get_item_ref(ComponentRef component,
     return ItemRef { item.vtable, reinterpret_cast<char *>(component.instance) + item.offset };
 }
 
+/// Convert a slint `{x: length, y: length, width: length, height: length}` to a Rect
```

Comment:
```suggestion
/// Convert a slint `{height: length, width: length, x: length, y: length}` to a Rect
```

Reply:
I thought that the function this comment applies to takes a tuple that's in height, width, x, y order. I know that the anonymous struct maps to such a tuple and inside the function body there's another comment, but I think that the earlier we remind the reader about this mapping the easier it is to read the code. Feel free to ignore though :)